The pull request's name

sijanec opened the pull request:
rok, pls, preveri, če dela ...

sijanec committed:
<i>probably</i> breaks the layout on small screens

sijanec committed:
podčrtica ni minus

sijanec committed:
krneki

sijanec committed:
multiple attached pictures, ability to remove them, storage bar >> msgz

sijanec committed:
meals almost done

sijanec committed:
fancy toast error messages

sijanec committed:
small fix

sijanec committed:
small fix 32

sijanec committed:
1.0.10-beta, hrana ~dela, netestirano, ker nimam menijev za izbrat

sijanec committed:
typo

sijanec committed:
brainfart

sijanec committed:
strpad > padstart

sijanec committed:
še enkrat

sijanec committed:
moram nehat commitat če ne dela, ane

sijanec committed:
some e2ee fixes

sijanec committed:
MIT baje ne potrebuje attributiona

sijanec committed:
meals fix and 1.0.11 version bump

sijanec committed:
meals submission fix (now sends all meals)

sijanec committed:
rewrite of meals.js

sijanec committed:
idk

sijanec committed:
med slovenscino

sijanec committed:
med slovenscino 2

sijanec committed:
added i18n, converted HTML files, TODO: js UI responses, TODO: translations
So basically, multilanguage support from an JavaScript object.
- only slovenian and english
- no change language UI yet
- only HTML strings are actually translated
  \- all text is translated, but javascript M.toast and other similar stuff does not pass trough s(), d(), S() and D() functions yet
- how to i18n:
	- add a string to js/lang/bundle.js TO ALL LANGUAGES (translation mgmt sys will have to be created)
	- HTML: create <x-??></x-??> tags, where 1st ? tells wether to just output the string (s) without dot suffix or (d) to try and append a . (for sentences and paragraphs) if it is
		not there yet. second ? tells wether to leave lowercase strings (l) or (u) to force the first character of the string be uppercase (capitalize()).
		-	put the key of your string as the innerHTML of the x-?? element.
  - JS: S("key") and D("key") return capitalized, s() and d() do not change case.

sijanec committed:
neki i18n js

sijanec committed:
deleting save files

sijanec committed:
all js files updated with i18n, not tested yet

sijanec committed:
deleted cache file

sijanec committed:
gitignore update for nano swp and save file

sijanec committed:
meals.js fix - still timing issues

sijanec committed:
meals.js fix - still timing issues

sijanec committed:
za narediti jezike

sijanec committed:
1.0.11.7

sijanec committed:
1.0.12-beta: podpora za jezike

sijanec committed:
1.0.12.28-beta: now auto update != all db clear

sijanec committed:
started rewriting gsec to JS

sijanec committed:
gsec

sijanec committed:
client side gimsisextclient

sijanec committed:
unfinished, don't touch

sijanec committed:
setup-storage critical fix (sw error) 1.0.12.35-beta

sijanec committed:
test ...

sijanec committed:
add server config for cors, fix gsec.js/postback

sijanec committed:
1.0.12.54-beta: gsec.js stuff

sijanec committed:
fix xss from gse

sijanec committed:
xss fropm gimsis fix

sijanec committed:
typo

sijanec committed:
1.0.12.59-beta: gsec.js fetchAbsences

sijanec committed:
gsec.js: fetchGrades, fetchMessagesLastPageNumber, fetchMessagesList

sijanec committed:
js/gsec.js: dodan fetchMessage, fixi na postbacku, dodatki
metode nimajo sprememb v apiju.

sijanec committed:
gsec.js fix

sijanec committed:
teachers.js, absences.js: uporabljajo gsec.js; gsec.js: spremembe apija

sijanec committed:
chats.js not done yet

sijanec committed:
DO NOT MERGE --- TESTING!

sijanec committed:
version bump (1.0.13-beta) because of jitsi

sijanec committed:
2nd version bump (1.0.13-beta) because of jitsi

sijanec committed:
DO NOT MERGE --- told ya u shouldn't

sijanec committed:
3rd DO NOT MERGE --- told ya u shouldn't

sijanec committed:
4rd DO NOT MERGE --- told ya u shouldn't

sijanec committed:
closer to merge, still don't

sijanec committed:
DNM; update

sijanec committed:
DNM; update

sijanec committed:
DNM; update

sijanec committed:
DNM; more update

sijanec committed:
DNM; more update

sijanec committed:
DNM; more update

sijanec committed:
absences: years old bug fixed

sijanec committed:
ready for merge?

sijanec committed:
added assetlinks.json for add

sijanec committed:
assetlinks more releases

rstular commented:
> post scriptum: aja zdej vidm... A ma github kakšno opcijo za joinat html fajle? A ni tako, da GitHub actions dela samo za pre-approved stvari, recimo PHP composer?
> 
> Plus po mojem je dobro imet narejene .html pages tudi v dev branchu, da se da dev branch v živo gledat na https://dev.gimb.tk/...

Fair point

rstular committed:
Styling changes, first round of bug fixes

There are (probably) more changes incoming, but we need to discuss them
first

rstular committed:
Minor styling fix

rstular commented on js/app.js:
```diff
@@ -5,8 +5,37 @@ if ("serviceWorker" in navigator) {
+    });
+}
+
+if (location.protocol != 'https:') {
+ location.href = 'https:' + window.location.href.substring(window.location.protocol.length);
+}
```
Tole sm vrgu vn, ker to itak dela nginx, men pa dela probleme v dev environmentu ker nimam certa na localhostu

rstular replied:
(torej no loger relevant)

rstular commented on js/messaging.js:
```diff
@@ -0,0 +1,511 @@
+        minLength: 0
+    });
+
+    if(window.location.hash.length > 1) {
+    	$("#full-name").val(decodeURIComponent(window.location.hash.substring(1)));
+    } else {
+    	$("#full-name").val(getUrlParameter("m"));
+    }
```
Je to backwards compatibility?
Bi se dalo uporabiti zgolj eno stvar (hash/param), ne obojega?

rstular committed:
Styling fixes, documentation

rstular commented on pages-src/jitsi.bvr:
```diff
@@ -0,0 +1,54 @@
+			</div>
+		</nav>
+		<@?i navigation@>
+		<!-- Maybe in the future? https://github.com/jitsi/jitsi-meet/blob/master/doc/api.md -->
```
V tem hipu irrelevant, jitsi ma svoj api, ce ga uporablas lahko prek JSa nastavlas display name userjem

rstular commented:
kako si GSE JS api testiru? Js mam s corsom probleme, lahko da mam v local environmentu kksno napako

rstular committed:
Code reformatting

rstular commented on js/gsec.js:
```diff
@@ -102,17 +102,20 @@ class gsec {
 					resolve(true);
 				} else {
-					if(!!(parsed.getElementById("lblMsg"))) { // če obstaja lblMsg (napaka pri prijavi)
+					if (!!(parsed.getElementById("lblMsg"))) { // če obstaja lblMsg (napaka pri prijavi)
```
A mas tuki namenoma 2 klicaja?

rstular committed:
Refactoring

sijanec commented:
Maš prav, lahko si pa tudi skopiraš wildcard iz serverja za testno upoarbo,
če hočeš. Se mi zdi da razni service workerji ne delajo brez HTTPS...

V V čet., 14. maj 2020 ob 02:00 je oseba rstular <notifications@github.com>
napisala:

> *@rstular* commented on this pull request.
> ------------------------------
>
> In js/app.js
> <https://github.com/beziapp/beziapp.github.io/pull/9#discussion_r424797320>
> :
>
> > +if (location.protocol != 'https:') {
> + location.href = 'https:' + window.location.href.substring(window.location.protocol.length);
> +}
>
> Tole bom vrgu vn, ker to itak dela nginx, men pa dela probleme v dev
> environmentu ker nimam certa na localhostu
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/beziapp/beziapp.github.io/pull/9#pullrequestreview-411378236>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKFM2EX34KVEMK3R66ZTCQTRRMYBLANCNFSM4M7XG7MA>
> .
>

sijanec commented:
Jap, backwards compatibility stvari, ki ni bila nikoli releasana (:.
Verjetno je hash bolje, ker ne podre cacha.

V V čet., 14. maj 2020 ob 02:19 je oseba rstular <notifications@github.com>
napisala:

> *@rstular* commented on this pull request.
> ------------------------------
>
> In js/messaging.js
> <https://github.com/beziapp/beziapp.github.io/pull/9#discussion_r424802576>
> :
>
> > +    if(window.location.hash.length > 1) {
> +    	$("#full-name").val(decodeURIComponent(window.location.hash.substring(1)));
> +    } else {
> +    	$("#full-name").val(getUrlParameter("m"));
> +    }
>
> Je to backwards compatibility?
> Bi se dalo uporabiti zgolj eno stvar (hash/param), ne obojega?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/beziapp/beziapp.github.io/pull/9#pullrequestreview-411384481>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKFM2EQ4G6KK2XGZVBKT3GTRRM2IFANCNFSM4M7XG7MA>
> .
>

sijanec commented:
Zanimivo, hvala!

>

sijanec commented:
Men dela. Tko da sem mel localdev-sijanec.gimb.tk s certom na
127.0.0.1, zgimsis.gimb.tk pa sem tistga na serverju uporabljal.

V V čet., 14. maj 2020 ob 03:12 je oseba rstular
<notifications@github.com> napisala:

>
> kako si GSE JS api testiru? Js mam s corsom probleme, lahko da mam v local environmentu kksno napako
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

sijanec commented:
Ne vem, sorry. Ne vem, zakaj the fucc sem to napisal...

V V čet., 14. maj 2020 ob 03:14 je oseba rstular <notifications@github.com>
napisala:

> *@rstular* commented on this pull request.
> ------------------------------
>
> In js/gsec.js
> <https://github.com/beziapp/beziapp.github.io/pull/9#discussion_r424816910>
> :
>
> >  					resolve(true);
>
>  				} else {
>
> -					if(!!(parsed.getElementById("lblMsg"))) { // če obstaja lblMsg (napaka pri prijavi)
>
> +					if (!!(parsed.getElementById("lblMsg"))) { // če obstaja lblMsg (napaka pri prijavi)
>
>
> A mas tuki namenoma 2 klicaja?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/beziapp/beziapp.github.io/pull/9#pullrequestreview-411401110>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKFM2ERCZHZPF3YWLDXEMWTRRNAWFANCNFSM4M7XG7MA>
> .
>

rstular committed:
Bug fix

sijanec committed:
bug fix fix

rstular committed:
:-)

rstular committed:
Bug fix ("|" isn't valid in a selector)

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

rstular committed:
Bug fix (Javascript is wierd)

rstular committed:
Attempt 1 at fixing encryption

rstular committed:
Logging/debugging code

rstular committed:
Fixed?

rstular committed:
Fix subject not showing

rstular commented:
> a ni tko da HTML5 podpira | v idju? http://xahlee.info/js/html_allowed_chars_in_attribute.html

idk, chrome se je men prtozvou, pa nism hotu prevec cajta zgublat

sijanec commented:
pametno

>

rstular committed:
Styling fixes

rstular commented:
> Zakaj pa nočeš `cache: false`?

Bil je debugging step, bom dal nazaj

rstular commented:
> pač lblmsg vsebuje "Napaka pri prijavi."

Porabu sm prevec cajta pr debugganju tega, sm biu ze na tem da bi reset --hard naredu hahaha

rstular commented on js/messaging.js:
```diff
@@ -0,0 +1,521 @@
+    setLoading(true);
+    // Load required data
+    let promises_to_run = [
+        localforage.getItem("username").then((value) => {
+            username = value;
+        }), localforage.getItem("password").then((value) => {
+            password = value;
+        }),
+    ];
+    Promise.all(promises_to_run).then(() => {
+        $.ajax({
+            url: API_ENDPOINT,
+            crossDomain: true,
+            data: {
+                "u": username,
+                "p": password,
+                "m": "izbrisisporocilo",
+                "a": id.replace(/_/g, "|")
+            },
+            dataType: "json",
+            cache: false,
+            type: "GET",
+            success: (data) => {
+                // If data is null, the request failed
+                if (data === null) {
+					UIAlert( `${D("unableToDeleteTheMessage")} ${D("requestFailed")}` );
+                } else {
+                    document.getElementById("msg_box-" + id).remove();
+                }
+                setLoading(false);
+            },
+
+            error: () => {
+				UIAlert( `${D("unableToDeleteTheMessage")} ${D("noInternetConnection")}` );
+                setLoading(false);
+            }
+
+        })
+    });
+}
```
Ta del kode uporablja star API, je to intentional?

rstular committed:
Revert debugging step

rstular committed:
Messaging redesign done

DO NOT MERGE - redesigning messages

CSS update

DNM - css update

DNM - hopefully last CSS update for now

DNM - I take it back, one more

DNM - debugging

DNM - Still debugging

DNM

DNM

DNM

DNM - WIP tab interface

DNM - better storage management

DNM - tabs update

DNM - messaging layout update

Messaging layout update

DNM - styling

DNM - Starting to add "compose functionality"

DNM - WIP FAB

DNM - WIP - modal not dismissible

DNM - Modal layout WIP

DNM - Encryption checkbox

DNM

DNM

DNM

DNM

DNM - Hopefully done?

DNM - Removed comments

rstular committed:
Moved storage notice

rstular committed:
Bug fixes

rstular committed:
Styling fixes

rstular committed:
Another styling fix

rstular commented:
Okej, at this point zadeva zgleda dost uredu.
Preden nardiva merge, bi blo fajn da pogledas kaj je z apijem v js/messaging.js, pa settings ne dela tko kt bi mogl, tkoda bi zaenkrat js to brisu iz navigationa. To je ubistvu pol to.

Meals sicer se nism meu cajta stestirat ker zdej ni obrokov, ampak whatever, do zacetka naslednga solskega leta se ta feature ne bo dost rabu.

sijanec committed:
it works on my machine -- messaging.js

sijanec committed:
e2ee msg error alert

sijanec committed:
always run the code before commiting

rstular committed:
Settings bug fix

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

sijanec committed:
zgimsis decrypt0r has no idea about THE REGEX

sijanec committed:
single byte commit

sijanec committed:
edit: nope, no need for regex .text() strips comments

rstular committed:
Styling fixes in about and settings

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

rstular committed:
Indentation fix

rstular committed:
Flag size fixes, hover

rstular committed:
Settings styling fix

sijanec committed:
fixed msg-compose-modal layout

sijanec committed:
Merge branch 'dev' of https://github.com/beziapp/beziapp.github.io into dev

sijanec committed:
version is now updated in sw.js and about.bvr

sijanec committed:
mesasging.js: deleteMsg spet dela

sijanec committed:
naredil skelet DB strukture messages (TypeError: Cannot read property '0' of null at messaging.js:129)

rstular committed:
Added gitignore

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

sijanec committed:
izgleda, da nisem prebral dokumentacije

sijanec committed:
Merge branch 'dev' of https://github.com/beziapp/beziapp.github.io into dev

sijanec committed:
well I could remember my own code but I didn't

sijanec committed:
idrk

rstular commented:
> kaj pomeni **/.commit?

File za git hook k ga mam postavlenga, git add mi ni delu v pre-commitu

rstular committed:
Indentation fix

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

rstular committed:
Remove beziapp references from chat

Lowers amount of data sent to the server

sijanec commented:
Drgač iirc maš `git commit -a`, ki avtomatsko nardi add, če to hočeš...

V V pet., 15. maj 2020 ob 00:35 je oseba rstular <notifications@github.com>
napisala:

> kaj pomeni **/.commit?
>
> File za git hook k ga mam postavlenga, git add mi ni delu v pre-commitu
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/beziapp/beziapp.github.io/pull/9#issuecomment-628921232>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKFM2EWJYQ7LRZ7G3YBDKJLRRRW3TANCNFSM4M7XG7MA>
> .
>

sijanec committed:
fix for my comment on 6624370

rstular commented:
> Se iskreno opravičujem, tab-size mam na 2 nastavljen in nism opazu, ni se mi pa dal VSC odpirat :D

No worries

rstular commented:
> Woah, a nas ne marajo?

Glede na to da so tebe blockal, ne zgleda da so ful fani

rstular commented:
Also, pustu sm ti readme na /home/debian/

sijanec committed:
removed BežiApp from chats sent to GimSIS

sijanec committed:
untested chats.js regex for chats in body

sijanec committed:
chats.js is just a useless piece of code

sijanec committed:
sendMessage gsec.js now requres and uses he.js

sijanec committed:
hmmmm, a je gimsis patchan za msg xss?

sijanec committed:
syntax error fix

sijanec committed:
DO NOT MERGE, testing new bVerbose

sijanec committed:
bvr (upstream) fixes, added support for nested commands
problem: bvr_compose_stream (upstream) still adds a bunch of newlines; i don't know where the fuxk they come from, so

<@?s latest_commit ?i .git/refs/heads/dev@> would still add some newlines around the included file, which is inacceptable.

but this is just a fix for the previous binary, that was just broken.

rstular committed:
Bug fix - handling afternoon lessons

sijanec committed:
testiram! DNM. btw, kako se naredi diff brez commita?

rstular committed:
GSEC bug fixes - handling commas properly

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

rstular committed:
Bug fix - absences time frame calculations

sijanec committed:
updating binaries from upstream (bverbose possible out of bounds a328bc0)

sijanec committed:
prev. commit in sw and about

rstular committed:
Linting

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

rstular committed:
Handlers moved from HTML to JS

sijanec committed:
dist should work, TODO: minify and bundle

rstular commented:
V nginx configu sm popravu `beziapp-dev` da ma document root v `/dist/`, to bo treba narest k bo sla zadeva v production

rstular committed:
Fix undefined variable in GSEC fetch timetable

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

rstular committed:
Deleting duplicate files

rstular committed:
Forgot to replicate changes from /assets to /dist

rstular committed:
Removed duplicate files

rstular committed:
Fixed install script

rstular committed:
Bug fix - GSEC regex matching

rstular committed:
GSEC bug fix - regex matching

rstular committed:
GSEC bug fix - fetch teachers

Subject with parenthesis in their name got improperly matched because of
a lazy flag. Now it's removed, it works properly.

sijanec committed:
tnx za popravljeno install skripto

sijanec committed:
popravljen dist, hvala rstular

rstular committed:
Modified language selection UI

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

rstular reviewed (commented):
No worries :-)

rstular committed:
Experimental theme support

sijanec committed:
missing loginError added to l18n

sijanec committed:
install

rstular committed:
Merge

rstular committed:
Bug fixes (settings, gsec)

rstular committed:
Regex fix

rstular committed:
Another regex bug fix

rstular committed:
Language changing bug fix

rstular committed:
CSS

sijanec committed:
changelog

sijanec committed:
fixed unreadable calendar text

sijanec committed:
fixed primary light on dark theme

sijanec committed:
forgot ./install

sijanec committed:
styling changes

rstular committed:
Bug fix attempt

sijanec committed:
CSS

sijanec committed:
merge

rstular committed:
Bug fix-timetable colors won't change from now on

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

rstular committed:
CSS bug fix

rstular committed:
Another CSS bug fix

rstular committed:
CSS fixes

rstular committed:
Theme fixes

rstular committed:
Styling bug fixes

rstular committed:
Styling fix, lopolis fix

rstular committed:
Revert "Styling fix, lopolis fix"

This reverts commit ea1ff86ecd12b698fe776801f87138221d01a9ce.

sijanec committed:
fixed ToS display

sijanec committed:
merge + css fix for tos

sijanec committed:
css .p => p

sijanec committed:
how'd that happen

sijanec committed:
how'd that happen -- ungitignored

sijanec committed:
does it work?

rstular committed:
Color fix

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

sijanec committed:
window.onerror

sijanec committed:
merge

sijanec committed:
install

rstular committed:
Bug fix - modal opens when hash is proivded

rstular committed:
CSS fixes

sijanec committed:
err reporting

sijanec committed:
fixed install script

sijanec committed:
did it work?

sijanec committed:
install scritp fix

sijanec committed:
#

sijanec committed:
DNM, not tested

sijanec committed:
DNM, not tested #2

sijanec committed:
DNM, not tested #3

sijanec committed:
DNM, not tested #3

sijanec committed:
fix for onunhandledrejection

sijanec committed:
made error reporting async

sijanec committed:
made error reporting async

sijanec committed:
this is not a fix

sijanec committed:
this is also not a fix

sijanec committed:
scope error?

sijanec committed:
scope error? + install

sijanec committed:
DNM

sijanec committed:
Revert "DNM"

This reverts commit 763061be7ee80fffd6dd1fd2bebbdacb2ad7183d.

sijanec committed:
Probi, hopefully bo to resl problen

sijanec committed:
maybe

sijanec committed:
changed load order

sijanec committed:
fixed geeky operators

sijanec committed:
replaced more geeky operators

sijanec committed:
fixed even mroe geeky operators (?)

rstular committed:
Styling fix

sijanec committed:
added teachers dir to dir

sijanec committed:
Merge branch 'dev' of https://github.com/beziapp/beziapp.github.io into dev

sijanec committed:
DO_NOT_MERGE; still testing error reporting optout

sijanec committed:
should be ok to merge

rstular committed:
Night theme

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

rstular committed:
Wrong theme name

sijanec committed:
latest_commit is not actually latest

rstular committed:
CSS fix

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

rstular committed:
CSS bug fix

rstular committed:
Another CSS bug fix

sijanec committed:
try-catch in fetchTeachers

sijanec committed:
try-catch in fetchTeachers (forgot instal)

rstular committed:
I don't know CSS :-(

rstular committed:
Revert "I don't know CSS :-("

This reverts commit 913b40fe12ea0f73614013fe5a528700d516749e.

rstular committed:
Install

rstular committed:
Indentation fix

sijanec committed:
made the night theme into a high contrast one

rstular committed:
New themes

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

rstular committed:
Bug fix

rstular committed:
Bug fix

sijanec committed:
version bump

sijanec committed:
Merge branch 'dev' of ssh://github.com/beziapp/beziapp.github.io into dev

sijanec committed:
version bunm + install

rstular committed:
Persist state

rstular committed:
Styling fix

sijanec committed:
testing minification with jsmin

sijanec committed:
Merge branch 'dev' of ssh://github.com/beziapp/beziapp.github.io into dev

sijanec committed:
minified their files from merge

sijanec committed:
forgot i'm not doing anything

rstular committed:
Changelog

rstular committed:
Merge branch 'dev' of github.com:beziapp/beziapp.github.io into dev

sijanec committed:
added pirati tema

sijanec committed:
Merge branch 'dev' of ssh://github.com/beziapp/beziapp.github.io into dev

rstular reviewed (approved):
Looks good
